### DISCUSSION TITLE
How to forecast new horizon on blender?

### DISCUSSION THREAD

**Discussion by georgepglv** [pycaret/pycaret#3446] [Q&A] [answered]
When doing an TSForecastingExperiment(), I can easily forecast any horizon I want using
`exp.predict_model(final_model, fh=270)`
if my final_model is a trained single model.

However, If it is a blender, it gives me: `ValueError: A different forecasting horizon `fh` has been provided from the one seen in `fit`. If you want to change the forecasting horizon, please re-fit the forecaster. This is because fitting of the `ForecastingPipeline` depends on `fh`. `

How can I forecast horizons different from my trained setup on blenders?

**Comment by ngupta23** [comment, marked as answer]
There is a bug for ensemble forecasters that needs to be fixed. See: https://github.com/pycaret/pycaret/issues/2329

**Reply by georgepglv** [reply]
oh, thanks. is there any workaround ?

**Reply by ngupta23** [reply]
Not for blend models. It works for individual models, so if you do not blend, it will work right now.

**Reply by georgepglv** [reply]
alright, thanks
